test/reload_test.go

```diff
@@ -101,8 +100,6 @@ func TestReloadMetricsHealth(t *testing.T) {
 	}
 	defer c1.Stop()
 
-	time.Sleep(100 * time.Millisecond)
```

Comment:
the sleep was added here... https://github.com/coredns/coredns/pull/1715
not obvious what the rationale was.